juego.py

```diff
@@ -238,11 +249,22 @@ def play(self, level):
     def run(self):
         pygame.init()
         self.screen = pygame.display.get_surface()
+
+        info = pygame.display.Info()
+
+        if not(self.screen):
```

Comment:
Why the parentheses around `self.screen`?

Reply:
Thanks for noticing! I actually added this code form one of the previous activities I have worked on. Nevermind, the redundant paranthesis are removed in d1e8746